Misc/NEWS.d/next/Documentation/2021-01-23-12-23-34.bpo-42948.apqCwM.rst

```diff
@@ -0,0 +1 @@
+Document bytearray.copy()
```

Comment:
```suggestion
Document ``bytearray.copy()``
```

Reply:
I am trusting that you are correct and that linking from blurbs works.  This also need a period.